src/aat/resources/features/F-1009 - Submit Case Creation Handle Case Links/F-1009_Test_Data_Base_Error_Response_V2.td.json

```diff
@@ -0,0 +1,26 @@
+{
+	"_guid_": "F-1009_Test_Data_Base_Error_Response_V2",
+
+	"productName": "CCD Data Store",
+	"operationName": "Submit case creation as Case worker",
+
+	"method": "POST",
+	"uri": "/case-types/{caseTypeId}/cases",
```

Comment:
If a **common TD file** to cover the basics of the API signature does not exist can we create on and use it as this endpoint is being used multiple times in these tests:

* F-1009_Test_Data_Base_Error_Response_V2
* F-1009_Test_Data_Base_V2

Reply:
Done [c129a9a](https://github.com/hmcts/ccd-data-store-api/pull/1872/commits/c129a9a6a6fdff4226e65e0b027691d3cd49352f)